src/database/mongoClient.js

```diff
@@ -0,0 +1,49 @@
+import { MongoClient } from 'mongodb';
+
+const MONGODB_URI = process.env.MONGODB_URI;
+
+// Database Name
+const dbName = 'cofacts';
+
+export default class CofactsMongoClient {
```

Comment:
Update: as [discussed in Slack](https://g0v-slack-archive.g0v.ronny.tw/index/channel/C2PPMRQGP/2020-05) we decided to add a comment to constructor to indicate that the users of `CofactsMongoClient` should never call `new` by themselves. 

When we adopt Typescript in the future, we will make the constructor private to make sure that users of `CofactsMongoClient` can only retrieve its singleton instance via `getInstance()` static method. `CofactsMongoClient` is dedicated to connecting to Cofacts LINE bot mongodb, not something that is generalized. ES6 class is used so that it's easier to define method and update the client type in the same time, also ensures a smoother updater when we rewrite in Typescript.